### PR TITLE
Updated the README with slightly clearer instructions on usage.

### PR DESCRIPTION
**Pull request by needcaffeine**
Not everyone realizes that they need to add the composer autoloader at the top of the page, so I added that to the usage. I also removed the testing section because there was no need for it. I added instructions to test the hello world application using PHP's inbuilt web server.

### CODE REVIEW

**Comment by mnapoli** [comment]
It seems you accidentally removed the `Testing` section.

**Comment by needcaffeine** [comment]
That was not an accident. I intentionally removed it. There are no tests when you start off with Slim. Anyone who knows how to write unit tests knows how to use phpunit. It's just confusing for newcomers.

**Comment by silentworks** [comment]
@needcaffeine thanks for your improvements to the README, but can you add the Testing section back please?

The Testing section does not confuse new users as new users would be pulling down Slim via composer or git clone the repo, the Testing section is necessary for contributors and there are already tests written for the framework itself.

**Comment by nunocodex** [comment]
+1 for restore the testing section, I think who working with the frameworks is a developer so know phpunit :+1:

**Comment by needcaffeine** [comment]
Okay, noted. I'll spend a little time making that section be more useful and restore it today. Thank you all for your thoughts on the subject.

**Comment by needcaffeine** [comment]
Here we go.

File: README.md [hunk 1]
```diff
@@ -1,56 +1,80 @@
...
-$app = new \Slim\App();
+<?php
+
+require 'vendor/autoload.php';
+
+$app = new Slim\App();
+
 $app->get('/hello/{name}', function ($request, $response, $args) {
     $response->write("Hello, " . $args['name']);
-    return $response;
```
Comment (akrabat) [inline README.md:28]: Best practice is to return the response. i.e. you are relying on an implementation detail of the current `Route` object if you don't return the `$response`.

Comment (needcaffeine) [inline README.md:28]: Hm, you're right.

Comment (needcaffeine) [inline README.md:28]: Fixed both in the README and in index.php.

**Comment by needcaffeine** [comment]
Is there anything further you'd like me to do to get this merged in?

**Comment by silentworks** [comment]
Thanks for this update @needcaffeine.

**Pull request merged**